### DISCUSSION TITLE
Desktop design not responsive to zoom

### DISCUSSION THREAD

**Discussion by cyrusae** [community/community#19139] [Accessibility]
For my experience (macOS Safari), it's impossible to make the text size on GitHub larger; cmd + a) makes the page wander off screen and b) causes some of the slowest text size increases while doing so I've ever seen, despite the UI *elements* (eg spacing) increasing rapidly and unhelpfully in size. 

Behavior in Google Chrome is different (last I checked, which was a couple weeks ago); text size increases, which causes UI elements to overlap one another and lose their text instead.

Problem: this is the case whilst I am trying to e.g. see the code review feedback on my pull requests whilst low vision.

**Comment by cehfisher** [comment]
Hello @cyrusae - I wanted to follow-up with you on this and ask if you are still seeing issues with browser zoom while using GitHub desktop. If yes, could you please share more details on which pages or features that have zoom issues? If your issue has been resolved, please close this ticket. Thanks! (cc @queenofcorgis)
